chart/app-example/templates/extendeddaemonset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: ExtendedDaemonSet
+metadata:
+  name: {{ include "app-example.fullname" . }}
+  labels:
+    {{- include "app-example.labels" . | nindent 4 }}
+spec:
+  strategy:
+    {{- toYaml .Values.strategy | nindent 4 }}
+  template:
+    spec: 
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "app-example.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      tolerations:
+      - operator: Exists
+      {{- toYaml . | nindent 6 }}
```

Comment:
I couldn’t find any englobing and not yet closed `with` statement.
This `.`  will pull all the variables, won’t it ?

Anyhow, the `- operator: Exists` toleration is already tolerating all the taints. So, there is no value in allowing the end-user to add some extra useless toleration.

```suggestion
```

Reply:
good catch